Bottom: a8cbdf2b9710c81703e1bfa15111d043ad7e9bff
Top:    6188860dfdb6e5b0d927f209eac2beab84ccc92d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 17:11:03 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b2539ca..7cdb152 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2686,7 +2686,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
